### DISCUSSION TITLE
Lxqt packages in flatpak

### DISCUSSION THREAD

**Discussion by telmobss1** [lxqt/lxqt#1951] [Ideas]
continuing the series of discussions about improvements, I would like to ask if it is worthwhile to package the applications in flatpak for two reasons 

Easier to distribute lxqt applications agnostic to interfaces

Easier for users to install and try out applications in beta or alpha, since each build is different, whereas in flatpak, the applications would be the same as any distro,Making debugging easier

What do you think?

**Comment by tsujan** [comment]
Negative.

Contrary to what may be advertised, things like flatpak can't work like standard installation and cause troubles. Users might think the bugs are in LXQt and report invalid issues. In short, everyone's time would be wasted.

**Reply by telmobss1** [reply]
yes, I know that flatpak is a double-edged sword, even though we know that flatpak applications are the future (we already have a lot of gtk and Qt software that are on the flathub and others that are in beta,such as discover and konsole)
In short, I would like to know how to design, think it is a good idea to package the applications of the Qt interface in flatpak (in testing mode)

**Reply by tsujan** [reply]
> we know that flatpak applications are the future

No, we don't. If they don't say "we're the future," who will? I've had enough of the troubles they cause for others.

They might make their methods better by experimenting but I don't think any LXQt developer has time for that.
